src/System.Text.Json/tests/Serialization/TestClasses.cs

```diff
@@ -898,26 +898,26 @@ public class BasicCompany : ITestClass
         public string name { get; set; }
 
         public static readonly byte[] s_data = Encoding.UTF8.GetBytes(
-            "{" +
-                @"""name"" : ""Microsoft""," +
-                @"""sites"" : [" +
-                    "{" +
-                        @"""street"" : ""1 Lone Tree Rd S""," +
-                        @"""city"" : ""Fargo""," +
-                        @"""zip"" : 58104" +
-                    "}," +
-                    "{" +
-                        @"""street"" : ""8055 Microsoft Way""," +
-                        @"""city"" : ""Charlotte""," +
-                        @"""zip"" : 28273" +
-                    "}" +
-                @"]," +
-                @"""mainSite"" : " +
-                    "{" +
-                        @"""street"" : ""1 Microsoft Way""," +
-                        @"""city"" : ""Redmond""," +
-                        @"""zip"" : 98052" +
-                    "}" +
+            "{\n" +
```

Comment:
What motivated this change?

Reply:
I wanted to verify that line numbers work properly.